[OCPBUGS-42978]: Updating etcd node scaling terminology (#83368)
openshift/openshift-docs: modules/etcd-node-scaling.adoc

[id="etcd-node-scaling_{context}"]
= Node scaling for etcd

> In general, clusters must have 3 control plane nodes. However, if your cluster is installed on a bare metal platform, you can scale a cluster up to 5 control plane nodes as a post-installation task. For example, to scale from 3 to 4 control plane nodes after installation, you can add a host and install it as a control plane node. Then, the etcd Operator scales accordingly to account for the additional control plane node.

Reviewer: Out of scope to your PR, and I can't even find a style guide ruling on this, but I know that our [post install section](https://docs.openshift.com/container-platform/4.17/post_installation_configuration/index.html) of the docs was somewhat recently reworded to use "postinstallation" as a single word. I don't like it personally, but changing it here would make it more consistent with our docs. Up to you:
```suggestion
In general, clusters must have 3 control plane nodes. However, if your cluster is installed on a bare metal platform, you can scale a cluster up to 5 control plane nodes as a postinstallation task. For example, to scale from 3 to 4 control plane nodes after installation, you can add a host and install it as a control plane node. Then, the etcd Operator scales accordingly to account for the additional control plane node.
```

Author: Good to know! I'll fix that.

> Scaling to 4-node or 5-node cluster control planes is available only on bare metal platforms.

Reviewer: I would suggest this phrase gets reworded a little bit, since it doesn't sound completely accurate as-is. Here are a few ideas:

```suggestion
Scaling to 4-node or 5-node control plane clusters is available only on bare metal platforms.
```

```suggestion
Scaling to 4-node or 5-node control planes is available only on bare metal platforms.
```

Reviewer: I see now that an SME suggested this original wording, but would my suggestions still be technically accurate? From a non technical perspective they read a little less awkwardly imo.

Author: Changed the wording as follows: "Scaling a cluster to 4 or 5 control plane nodes is available only on bare metal platforms."

Reviewer 2: "Scaling a cluster to 4 or 5 control plane nodes is available only on bare metal platforms." is great - thanks.

"Control plane clusters" would mean low-level clusters that implement the control plane of the OCP cluster. There is the etcd cluster; few other control plane components could be considered as forming clusters of their own. While technically accurate in this sense, I don't remember seeing any materials that would refer to these components as clusters, with the exception of the etcd cluster.

This paragraph is talking about the number of control plane nodes in an OCP cluster. 

Hope this helps.

Author: Thanks for the explanation!

> For more information about how to scale from 3 to 4 control plane nodes by using the Assisted Installer, see "Adding hosts with the API" and "Installing a primary control plane node on a healthy cluster".

Reviewer 2: Since we're touching this, and assuming it applies to 5 control plane nodes, too, how about

```suggestion
For more information about how to scale from 3 to 4 or 5 control plane nodes by using the Assisted Installer, see "Adding hosts with the API" and "Installing a primary control plane node on a healthy cluster".
```
